docs/atlascli/command/atlas-alerts-settings-list.txt

```diff
@@ -37,6 +37,10 @@ Options
      - Type
      - Required
      - Description
+   * - -c, --compact
```

Comment:
--compact as name and description suggests feature but in reality is introduced only to support backward compatibility. Names are hard but if we can add something more descriptive:

--legacy-output
--old-format 

Etc.

Reply:
Sure I'm happy to go with that

Reply:
Will ask others for opinion before applything this comment. (Names are hard)

Reply:
Chatted offline. Compact seems much better as we do not use old SDK and this flag can stay with us without removal